src/services/accessValidation.js

```diff
@@ -76,17 +87,6 @@ export async function setReadRegions(query, userId, useFirstReadRegion = false)
 
   // if region.in is part of query (user has requested specific regions)
   if ('region.in' in query && Array.isArray(query['region.in']) && query['region.in'][0]) {
-    // first check to see if "all regions (central office)" is selected
-    // if so, return all regions has access to
-
-    if (query['region.in'].length === 1 && parseInt(query['region.in'][0], DECIMAL_BASE) === 14) {
```

Comment:
This change causes central office users to see no reports/data on the landing page.

Reply:
I'm confused by this - I'm looking at the site as a Central office user and am seeing plenty of data. Is it that central office users do not have permissions to any regions on the live app?

Reply:
Region in query params is `region.in[]=14`. The user is central office with permissions on region 1 and 3.

Reply:
There won't be a region.in[]=14 in the query any more, as far as I know.

The way it's coded now, the user above should see data for 1 & 3 unless they've filtered by region.
Is this correct? Should they instead see all region data?

Reply:
I still get `regionId[]=14` locally on this branch on the landing page

Reply:
ah gotcha, the **landing page**, my bad